src/SonarScanner.MSBuild.TFS/CoverageReportProcessorBase.cs

```diff
@@ -64,24 +64,38 @@ public bool ProcessCoverageReports()
 
             Debug.Assert(this.config != null, "Expecting the config to not be null. Did you call Initialize() ?");
 
-            // Fetch all of the report URLs
-            Logger.LogInfo(Resources.PROC_DIAG_FetchingCoverageReportInfoFromServer);
-
-            if (TryGetTrxFiles(this.config, this.settings, out var trxPaths) &&
-                trxPaths.Any() &&
-                config.GetSettingOrDefault(SonarProperties.VsTestReportsPaths, true, null) == null)
+            if (config.GetSettingOrDefault(SonarProperties.VsTestReportsPaths, true, null) != null)
```

Comment:
There needs to be a regression test for this change (no need to cover all cases, just the case that we shouldn't search if the property is provided).

Reply:
Tests has been added.